components/thumbnail/stories/thumbnail.test.js

```diff
@@ -15,13 +15,30 @@ export const ThumbnailGroup = Variants({
 		{
 			testHeading: "With background",
 			backgroundColor: "orange",
+		},
+		{
+			testHeading: "Image fit - cover",
+			imageURL: "example-card-landscape.png",
+			isCover: true,
+		},
+		{
+			testHeading: "Image fit - landscape",
+			imageURL: "example-card-landscape.png",
+		},
+		{
+			testHeading: "Image fit - portrait",
+			imageURL: "example-card-portrait.png",
 		}
 	],
 	stateData: [
 		{
 			testHeading: "Disabled",
 			isDisabled: true,
 		},
+		{
+			testHeading: "Focused",
+			isFocused: true,
+		},
```

Comment:
I would also add coverage for focused + selected on layer: https://pr-3087--spectrum-css.netlify.app/preview/?path=/story/components-thumbnail--default&args=isLayer:!true;isSelected:!true;isFocused:!true